### PR TITLE
Addon-viewport: Add viewports of the latest iPhones

### PR DESCRIPTION
**Pull request by YutaSugimura**
The addition of new devices

What was done:
* Added iPhone12, iPhone12 mini, iPhone12 Pro Max, iPhone SE2.

### CODE REVIEW

**Review by shilman — APPROVED** [review]
LGTM @yutasugimura. thanks for your contribution!!

File: addons/viewport/src/defaults.ts [hunk 1]
```diff
@@ -57,6 +57,38 @@ export const INITIAL_VIEWPORTS: ViewportMap = {
...
+  iphone12: {
+    name: 'iPhone 12',
+    styles: {
+      height: '844px',
+      width: '390px',
+    },
+    type: 'mobile',
+  },
+  iphone12promax: {
+    name: 'iphone 12 Pro Max',
```
Comment (gaetanmaisse) [inline addons/viewport/src/defaults.ts:85]: `iphone` should have a capitalized `P` like the other ones

**Review by gaetanmaisse — APPROVED** [review]
@YutaSugimura I just added a mini-comment for a typo, can you take in into account and ping me so that I merge the PR?

File: addons/viewport/src/defaults.ts (hunk 1 above)
Comment (YutaSugimura) [reply addons/viewport/src/defaults.ts]: Thanks for letting me know about the typo mistake.
It has been fixed.

**Comment by gaetanmaisse** [comment]
Thanks for your contribution @YutaSugimura!

**Pull request merged**